lua/core/autopairs.lua

```diff
@@ -1,54 +1,63 @@
--- if not package.loaded['nvim-autopairs'] then
---   return
--- end
-local Log = require "core.log"
-local status_ok, _ = pcall(require, "nvim-autopairs")
-if not status_ok then
-  Log:get_default().error "Failed to load autopairs"
-  return
+local M = {}
+
+function M.config()
+  lvim.builtin.autopairs = {
+    active = true,
+    ---@usage  map <CR> on insert mode
+    map_cr = true,
+    ---@usage auto insert after select function or method item,
+    map_complete = vim.bo.filetype ~= "tex",
```

Comment:
Probably out of scope for this PR, but this only works if the first file I open is a `tex` file.
Should we make this a function and call `require("nvim-autopairs.completion.compe").setup` when the file is changed ? IDK, in the meantime maybe add a note.